.github/workflows/ci.yml

```
@@ -2,127 +2,421 @@ name: CI
-          node-version: ${{ env.NODE_VERSION }}
-
-      # Setup Bun with specific version
       - name: Setup Bun
         uses: oven-sh/setup-bun@v1
         with:
           bun-version: 1.0.14
 
-      # Cache Bun dependencies
+      - name: Generate cache key
+        id: cache-key
+        run: echo "value=${{ runner.os }}-bun-${{ hashFiles('**/bun.lockb') }}" >> $GITHUB_OUTPUT
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:1:71: Double quote to prevent globbing and word splitting

(shellcheck)